codegenerator/integration_tests/tests/test_indexers/test_exits/pnpm-lock.yaml

```diff
@@ -1515,6 +1519,10 @@ snapshots:
     dependencies:
       once: 1.4.0
 
+  envio@file:../../../../cli/npm/envio:
+    dependencies:
+      rescript: 11.1.3
+
```

Comment:
For comparison, `link:` didn't the rescript dependency in the pnpm-lock. I think this is what caused the issue with not being able to find the rescript dependency.